opentech/static_src/src/app/src/containers/Note.js

```diff
@@ -18,10 +17,12 @@ class Note extends React.Component {
     render() {
         const { note } = this.props;
 
+        const date = new Date(note.timestamp).toLocaleDateString('en-gb', {day: 'numeric', month: 'short', year:'numeric'})
```

Comment:
So we are going to have to do it for any date separately? We probably should create a reusable function. Also, shouldn't we specify `timeZone: 'GMT'`?

Reply:
So the datetime string coming from django is timezone aware. But i think you are correct the brower might localise it. Lets make sure

Reply:
Yes, browser should use the time zone set in the OS if the date object is time zone-aware.